src/sentry/integrations/bitbucket/client.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import absolute_import
+
+
+from six.moves.urllib.parse import urlparse
+
+from sentry.utils.http import absolute_uri
+from sentry.integrations.atlassian_connect import integration_request
+
+BITBUCKET_KEY = '%s.bitbucket' % urlparse(absolute_uri()).hostname
+
+
+class BitbucketAPI(object):
```

Comment:
Can this extend the integrations API client? /cc @macqueen

Reply:
@EvanPurkhiser yes it can. I have another PR https://github.com/getsentry/sentry/pull/8482 where I do that. This is just for testing that the JWT stuff worked. I can remove it from this PR if that makes more sense.